langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaClient.java

```diff
@@ -29,23 +30,34 @@
 import static com.google.gson.FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES;
 import static java.lang.Boolean.TRUE;
 
+@Slf4j
 class OllamaClient {
 
     private static final Gson GSON = new GsonBuilder()
             .setFieldNamingPolicy(LOWER_CASE_WITH_UNDERSCORES)
             .create();
 
     private final OllamaApi ollamaApi;
+    private final boolean logStreamingResponses;
 
     @Builder
     public OllamaClient(String baseUrl,
                         Duration timeout,
+                        Boolean logRequests, Boolean logResponses, Boolean logStreamingResponses,
```

Comment:
can't `logResponses` be re-used for streaming (instead of introducing `logStreamingResponses`)?